### PR TITLE
PR for today's session

### PR DESCRIPTION
**Pull request by jread-usgs**

### CODE REVIEW

File: content/r-package-dev/Maintenance.Rmd [hunk 1]
```diff
@@ -50,29 +50,32 @@ So, what are you really on the hook for when creating a package? It depends on w
...
+If you are having trouble motivating yourself to fill in documentation and create tests because you are the only user of the package you are writing, remember that you are supporting future you. Future you doesn't remember the assumptions that you made when writing that package, and future you wishes you had dropped a few hints as to what that clever chunk of code was supposed to do. The "Personal use" user group is noted here because it is common (turning your scripts into a cohesize package gives you access to all of those general benefits of R packages), but our advice is to design and maintain a package for yourself in an identical fashion to supporing several knowledgeable users.
```
Comment (wdwatkins) [inline content/r-package-dev/Maintenance.Rmd]: typo in the last sentence

File: content/r-package-dev/Maintenance.Rmd [hunk 2]
```diff
@@ -148,16 +151,63 @@ summarize_sim(x=c(4,5), y=c(0,89), fig_path = '../my_figure.png')
 
 ### Define *how* to engage with package developers
 e.g., github, email, other
+![Labeling github issues](../static/img/maintenance_labeled_issues.png#inline-img "using labels for github issues")
 
 
 ## Maintaining your R package
 
-From GH issue: **THIS IS PLACEHOLDER TEXT**
+From GH issue: **THIS IS PLACEHOLDER TEXT**
```
Comment (lindsaycarr) [inline content/r-package-dev/Maintenance.Rmd]: Still want this?

File: content/r-package-dev/Maintenance.Rmd [hunk 3]
```diff
@@ -148,16 +151,63 @@ summarize_sim(x=c(4,5), y=c(0,89), fig_path = '../my_figure.png')
...
  * Use reviewer requests on GitHub.
- * Describe process to update packages on GRAN.
+ * Updating packages on GRAN.
+ 
+### Identify requirements of package maintenance
+How do you decide on maintenance priorities? Unless you have a full time job to support a single package, you will often have to 
+make hard decisions about what changes to prioritize over others. Priorities are decided in a number of different ways, and package 
+developers and maintainers have different roles in that prioritization. In some cases, we take cues from a committee that decides what is 
+important, and in other cases a single individual has full say over what the maintenance priorities are. Having a blended role 
+can work out best, where there is oversight provided via collaboration, but the maintainer still have a clear seat at the table. Ultimately,
```
Comment (lindsaycarr) [inline content/r-package-dev/Maintenance.Rmd]: have --> has

**Comment by lindsaycarr** [comment]
Content for #157

**Pull request merged**